frontend-service/src/test/java/com/amazonaws/blox/frontend/integration/DeleteEnvironmentIntegrationTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.frontend.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.amazonaws.blox.dataservicemodel.v1.model.Environment;
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentHealth;
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentType;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DeleteEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DeleteEnvironmentResponse;
+import com.amazonaws.blox.frontend.operations.DeleteEnvironment;
+import com.amazonaws.serverless.proxy.internal.model.AwsProxyRequest;
+import com.amazonaws.serverless.proxy.internal.model.AwsProxyResponse;
+import com.amazonaws.serverless.proxy.internal.testutils.AwsProxyRequestBuilder;
+import java.time.Instant;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+public class DeleteEnvironmentIntegrationTest extends IntegrationTestBase {
+
+  private Environment environment;
+
+  @Before
+  public void setup() {
+    environment =
+        Environment.builder()
+            .environmentId(environmentId)
+            .role("role")
+            .environmentType(EnvironmentType.Daemon)
+            .environmentHealth(EnvironmentHealth.HEALTHY)
+            .environmentStatus("ok")
+            .deploymentMethod("TerminateThenReplace")
+            .createdTime(Instant.now())
+            .lastUpdatedTime(Instant.now())
+            .build();
+  }
+
+  @Test
+  public void testDeleteEnvironment() throws Exception {
+    when(dataService.deleteEnvironment(any()))
+        .thenReturn(DeleteEnvironmentResponse.builder().environment(environment).build());
+
+    final AwsProxyRequest request =
+        new AwsProxyRequestBuilder("/v1/myCluster/environments/myEnv", "DELETE").build();
```

Comment:
Hm. Yeah, I'm not convinced that we want this layer of testing for every API. I think it's okay to merge in as a starting point, but we should consider turning this integration test into a functional test of just the FE behaviours.

Reply:
Sure. I will merge this one, and let's update the test later per our integration discussion.